crates/uv-resolver/src/resolution.rs

```diff
@@ -714,6 +728,14 @@ impl std::fmt::Display for DisplayResolutionGraph<'_> {
                 // Write the line as is.
                 writeln!(f, "{line}")?;
             }
+
+            if self.include_indices && node.index().is_some() {
```

Comment:
should this conditional include `self.include_annotations`?

Reply:
I think it's okay as-is.